### PR TITLE
Add SE-0269 to the changelog

### PR DESCRIPTION
**Pull request by Jumhyn**
This updates the changelog with the changes introduced by SE-0269.

Implemented by #23934.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,33 @@ CHANGELOG
 
 </details>
 
+Swift Next
+----------
+
+* [SE-0269][]:
```
Comment (theblixguy) [inline CHANGELOG.md]: Please add this to the bottom of this file (below the last SE):

```
[SE-0269]: <https://github.com/apple/swift-evolution/blob/master/proposals/0269-implicit-self-explicit-capture.md>
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,33 @@ CHANGELOG
 
 </details>
 
+Swift Next
```
Comment (theblixguy) [inline CHANGELOG.md]: Please add this to the table.

**Comment by Jumhyn** [comment]
@theblixguy Updated per your comment. I received an email with a comment about adding "Swift Next" to the table at the top, but don't see that comment now--is that still relevant?

**Comment by theblixguy** [comment]
I thought I remembered seeing it in the past but I went through the diff but couldn’t find it!

**Comment by theblixguy** [comment]
@swift-ci please smoke test

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,33 @@ CHANGELOG
...
+----------
+
+* [SE-0269][]:
+
+  When an escaping closure explicitly captures `self` it its capture list, the
+  use of implicit `self` is enabled within that closure. This means that the
+  following code is now valid:
+  
+  ```
```
Comment (theblixguy) [inline CHANGELOG.md]: Just noticed - please could you change it to this:

```suggestion
  ```swift
```

so the code gets syntax-highlighted.

**Comment by Jumhyn** [comment]
@theblixguy Heads up that I've added the `swift` annotation to the code block here—this should be ready.

**Comment by theblixguy** [comment]
@swift-ci please smoke test

**Review by theblixguy — APPROVED** [review]
LGTM, but I'll tag someone from Apple to take a look as well.

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,6 +23,33 @@ CHANGELOG
 
 </details>
 
+Swift Next
+----------
+
+* [SE-0269][]:
+
+  When an escaping closure explicitly captures `self` it its capture list, the
```
Comment (brentdax) [inline CHANGELOG.md]: Nit: should be “in its capture list”, not “it its”

**Review by beccadax — CHANGES_REQUESTED** [review]
Just a typo.

File: CHANGELOG.md (hunk 4 above)
Comment (theblixguy) [reply CHANGELOG.md]: Ugh I completely missed this! Thank you for spotting it!

Comment (Jumhyn) [reply CHANGELOG.md]: 😱 good catch, thanks Brent!
```suggestion
  When an escaping closure explicitly captures `self` in its capture list, the
```

**Review by beccadax — APPROVED** [review]

**Comment by brentdax** [comment]
@swift-ci please smoke test

**Comment by Jumhyn** [comment]
@brentdax @theblixguy Could one of you merge this?

**Comment by Jumhyn** [comment]
Thanks!

**Pull request merged**